### ISSUE TITLE
Intermediate component for bundling archiver data into a Run

### ISSUE THREAD

**Issue by danielballan** [bluesky/databroker#567]
Prompted by NSLS-II Users' Meeting discussion

**Comment by mrakitin** [comment]
It would probably be nice to have the archiver data bundled into the exported runs with https://github.com/bluesky/databroker-pack, so that a separate, external data source.

Discussed in a call with @ambarb, @tankonst, and @lwiegart.

**Comment by danielballan** [comment]
If we can get Archiver properly re-integrated with databroker v1/v2, `databroker-pack` will "just work" and treat the Archiver time series as just another data stream. Let's discuss how to implement that.

**Comment by ambarb** [comment]
But in that case, you would have to pre-declare which PVs you wanted packed, right?  Not a show stopper, just clarifying question

**Comment by mrakitin** [comment]
@ambarb, with the current implementation of the archiver appliance event source in databroker, one has do declare the PVs via a configuration, or dynamically: https://blueskyproject.io/databroker/v1/archiver.html#configuration. Right now the question is about how to properly insert the data from the archiver into databroker during the collection time (or somehow else).

**Comment by ambarb** [comment]
Right, but you don't want to insert all of them at the time of package?  this is probably overkill and too resource heavy?

Either way, I vote for a separate stream.  Archiver time stamp has a +/- factor for muxing with data.

**Comment by mrakitin** [comment]
The archiver feature has not been heavily tested in production yet, so we need to learn about specific use cases. I know CHX and ISS were looking at it, and @elistavitski even has a GUI to visualize the archiver data via the databroker API, but we haven't explored behind this. Let's have a discussion early next week between @danielballan, @ambarb and me to see what the requirements are and how to implement it.

**Comment by ambarb** [comment]
In our offline discussion, we reviewed some existing options.  With some further investigation, what is returned to by databroker is a pandas data type and format is like that of each point collected by Bluesky when returned by databroker (`header.table()['time']`).  Currently, this is a human readable time in EST, not in seconds.  For some applications, only the relative seconds matter.  If time-sensitive applications, there is either a timestamp, or the data is collected in "1 point".  SO to overlay with some data, we need to have the option to return at minimum, the epoch value (same format and reference as `header.start.time`).   We should have an option on the format we want returned is all that I am saying.
